modules/edns_keepalive/edns_keepalive.c

```diff
@@ -48,7 +48,7 @@ static int edns_keepalive_finalize(kr_layer_t *ctx)
 	return ctx->state;
 }
 
-KR_EXPORT int edns_keeapalive_init(struct kr_module *self)
+KR_EXPORT int edns_keepalive_init(struct kr_module *self)
```

Comment:
api break

Reply:
This doesn't surprise me, it's actually fairly typical IME (and you can imagine I have a lot of experience w/ this) to have 1 or 2 useful finds.